base/freestanding/femr/config/keycloak/realm-data.json

```diff
@@ -3779,6 +3779,94 @@
     ],
     "adminEventsEnabled": true,
     "adminEventsDetailsEnabled": false,
+    "identityProviders": [
+      {
+        "alias": "uw-netid",
+        "displayName": "UW NetID",
+        "internalId": "02474217-3843-4206-9b37-b5d392d2a8a9",
+        "providerId": "saml",
+        "enabled": false,
+        "updateProfileFirstLoginMode": "on",
+        "trustEmail": true,
+        "storeToken": false,
+        "addReadTokenRoleOnCreate": false,
+        "authenticateByDefault": false,
+        "linkOnly": false,
+        "firstBrokerLoginFlowAlias": "first broker login",
+        "config": {
+          "validateSignature": "true",
+          "signingCertificate": "MIIDuTCCAqGgAwIBAgIJAKJ/HGIkuxTwMA0GCSqGSIb3DQEBCwUAMHMxCzAJBgNV\nBAYTAlVTMQswCQYDVQQIDAJXQTEVMBMGA1UEBwwMRGVmYXVsdCBDaXR5MSEwHwYD\nVQQKDBhVbml2ZXJzaXR5IG9mIFdhc2hpbmd0b24xHTAbBgNVBAMMFGlkcC51Lndh\nc2hpbmd0b24uZWR1MB4XDTIxMDMyOTIwNTg1MloXDTQxMDMyNDIwNTg1MlowczEL\nMAkGA1UEBhMCVVMxCzAJBgNVBAgMAldBMRUwEwYDVQQHDAxEZWZhdWx0IENpdHkx\nITAfBgNVBAoMGFVuaXZlcnNpdHkgb2YgV2FzaGluZ3RvbjEdMBsGA1UEAwwUaWRw\nLnUud2FzaGluZ3Rvbi5lZHUwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIB\nAQDB/RvJuvC9B3/W5n+P+3PhEcYA0K26udDX0otmE0FnNb3lCD2G2ctWoEjbnOHc\nJ90bQtLjkG3TU2FAIlU3GIGmHHiVQe7Mln/UMhiK4JWTzLvfHOYVm+76sL7Ond7N\nkpu7QYm9r+sUAw4meCicaXQL7KD0JZhnYKRX5gLImK52BVy0x4G4AnP2IVfJbm02\npFHSN2zGFn4fXZI/g1RlJrdoC7UFM27h/BFOOL514K0GElzUJJkXMqBN64+EYYW7\nv6eQ9z/p7jrpC2BTi0WjXoohsk7tz+4LXP3cv1I1K3/qR++VFFOy29+vXRhH2cL1\nbSE17HcqdKQX7p+Aj/NPB/TdAgMBAAGjUDBOMB0GA1UdDgQWBBQ/mybHdlgo4MyQ\nOCWRNt268twZgDAfBgNVHSMEGDAWgBQ/mybHdlgo4MyQOCWRNt268twZgDAMBgNV\nHRMEBTADAQH/MA0GCSqGSIb3DQEBCwUAA4IBAQA0hM+G2sfyL6FHZNI0nTCIjVDi\nUvDnacpQgnpEvsS7l/6xQvuOwjeRxk66rpBMk+9T/Mk1AoNCqi4YP51JFHGYP+iu\nRosS6up+usX/tNgg5QhZwBaV+KXOxxVOuvGH5GnPLJmp7lMBBDsW9/LYMrLPX42L\nwDUOJ5rTF1PjnqdLEZVtXPYCJIiPrzkvQCY5YjcwHmZO4mw7F2HaqT+w3bCf7UWl\nPPKWoBBqR7kfH2WN2PJw/FhHU6XbBU2n4/RwRTZ3NwtY75mahb1dz2zyMbavAHN2\nRQ8TBe5L97w84ZZaoCK7laQgFczmMucg1JxBtU7jifO/ew4b/9zQ7JyggkYm",
```

Comment:
any risk associated with keeping this in the repository?

Reply:
@pbugni @mcjustin 


>any risk associated with keeping this in the repository?

Yes. If UW IAM updates [their metadata endpoint](https://idp.u.washington.edu/metadata/idp-metadata.xml) with a new cert, we would need to update this value. 

The current cert is set to expire in 20 years, so unless it's compromised, it's unlikely we would need to update this value.

---
The [Use SAML 2.0 for single sign-on page](https://itconnect.uw.edu/security/iam/sso/saml/) page, in the **Other operational practices** section as some more info:
>**Identity provider key rollovers**
>Occasionally, we may need to change the certificate we publish in the UW IdP metadata. This is known as a “key rollover”. It’s the process by which our current certificate is replaced by another certificate. If your SAML software supports multiple certificates (signing keys) for a single entity in IdP metadata, the transition may be seamless. If not, we will rely on the contact information provided during service provider registration to minimize service disruption during a key rollover.

Reply:
Marking this as resolved for now, please un-resolve if necessary